src/controller/index.js

```diff
@@ -26,5 +26,4 @@ router.post('/login', loginValidation, whoExist, home.loginPost);
 
 router.get('/userPrevQ', getToken, renderAllPosts, userPrevQ.get);
 router.get('/userMyProfile', getToken, renderMyPosts, userMyProfile.get);
-router.post('/signup', signupValidation, home.signupPost);
```

Comment:
@naremanhilles Did you mean to remove this signup post handler?

Reply:
yes becaues the same rout repeats at the same file :+1:

Reply:
Ah, got it!